Library/Formula/glibc.rb

```diff
@@ -19,6 +21,9 @@ def install
         "--prefix=#{prefix}",
         "--enable-obsolete-rpc",
         "--without-selinux"] # Fix error: selinux/selinux.h: No such file or directory
+      unless OS.mac?
```

Comment:
Please fix it up. Make your changes, then commit them using `git commit --amend Library/Formula/glibc.rb` and push them using `git push --force`. Your pull request will be updated.